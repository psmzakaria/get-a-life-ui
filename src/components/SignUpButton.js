import React from "react";
import { Form, Checkbox, Button, Divider } from "semantic-ui-react";

const SignUpButton = props => {
  return (
    <div>
      <Form.Field>
        <Checkbox label="I agree to the Terms and Conditions" />
      </Form.Field>
      <Button
        fluid
        id="signup"
        color="blue"
        type="submit"
        onClick={props.handleOnSubmit}
      >
        Submit
      </Button>
      <Divider />
      <div>
        <p className="text-align-center">
          Have an account?
          <a
            className="cursor-pointer"
            onClick={() => props.loadComponent("Sign In")}
          >
            {` `} Sign In Here
          </a>
        </p>
      </div>
    </div>
  );
};

export default SignUpButton;
