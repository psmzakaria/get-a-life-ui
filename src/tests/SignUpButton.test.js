import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import SignUpButton from "./../components/SignUpButton";

it("should render the sign up button ", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<SignUpButton />);
  const output = renderer.getRenderOutput();

  expect(output).toMatchSnapshot();
});
