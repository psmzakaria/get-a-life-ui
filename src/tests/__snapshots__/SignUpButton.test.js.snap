// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the sign up button  1`] = `
<div>
  <Button
    as="button"
    color="blue"
    fluid={true}
    id="signup"
    onClick={undefined}
    role="button"
    type="submit"
  >
    Submit
  </Button>
  <Divider />
  <div>
    <p
      className="text-align-center"
    >
      Have an account?
      <a
        className="cursor-pointer"
        onClick={[Function]}
      >
         
         Sign In Here
      </a>
    </p>
  </div>
</div>
`;
